src/main/java/org/broadinstitute/hellbender/tools/HaplotypeCallerSpark.java

```diff
@@ -267,13 +293,24 @@ private void writeVariants(JavaRDD<VariantContext> variants) {
             final Broadcast<ReferenceMultiSource> reference,
             final Broadcast<HaplotypeCallerArgumentCollection> hcArgsBroadcast,
             final ShardingArgumentCollection assemblyArgs,
-            final SAMFileHeader header) {
+            final SAMFileHeader header,
+            final Broadcast<VariantAnnotatorEngine> annotatorEngineBroadcast) {
```

Comment:
Thanks, that basically what I figured but I just wanted to hear the explanation to be sure.